src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,92 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import * as React from 'react';
+import { useState } from 'react';
+import { actionContainer } from '../Notification/Notification.style';
+import {
+  toastContainer,
+  topContainer,
+  infoContainer,
+  infoIconContainer,
+  actionIconsContainer,
+  chevronIconContainer,
+  expandedContainer,
+} from './Toast.style';
+import { typeToIconName } from '../Notification/subcomponents/CompactNotification/CompactNotification';
+import Icon from '../Icon';
+import { NotificationTypes } from '../Notification/Notification';
+import { AcceptedColorComponentTypes } from '../../utils/themeFunctions';
+import { TestId } from '../../utils/types';
+import { generateTestDataId } from '../../utils/helpers';
+
+export type ToastType = AcceptedColorComponentTypes | string;
+
+export type Props = {
+  /** The informative message of the Toast */
+  message: string;
+  /** The type of the Toast, will determine the color and the icon */
+  type: ToastType;
```

Comment:
why this is `AcceptedColorComponentTypes | string` ?

Reply:
my initial thought was that the user may want to use a Toast type different than the ones in AcceptedComponentTypes (for now only AcceptedComponentTypes are accepted) and therefore a default color (darkGray) is set for this (designs also include a default color/no icon version of the toast)

Reply:
there is no other type though so i would stick with what we know so no string for now